public/client-v2/src/actions/FrontsConfig.js

```diff
@@ -0,0 +1,38 @@
+// @flow
+
+import type { Action } from '../types/Action';
+import type { FrontsConfig } from '../types/Fronts';
+
+import fetchFrontsConfig from '../services/faciaApi';
+
+function frontsConfigReceived(config: FrontsConfig): Action {
+  return {
+    type: 'FRONTS_CONFIG_RECEIVED',
+    payload: config
+  };
+}
+
+function requestFrontsConfig(): Action {
+  return {
+    type: 'FRONTS_CONFIG_GET_RECEIVE',
+    receivedAt: Date.now()
+  };
+}
+
+function errorReceivingConfig(error: string): Action {
+  return {
+    type: 'CAUGHT_ERROR',
+    message: 'Could not fetch fronts config',
+    error,
+    receivedAt: Date.now()
+  };
+}
+
+export default function getFrontsConfig() {
+  return (dispatch: Dispatch) => {
```

Comment:
I wonder if this code is getting covered with just this `Dispatch` type? Not the end of the world but may need to type it with a `Thunk`. Although I think the `Thunk` type that I added into `types/Stores.js` might be unfit for purpose. Something like the below might do it:

```
type ThunkDispatch = (action: ThunkAction) => void;

export type Store = ReduxStore<State, Action>;
export type GetState = () => State;
export type Dispatch = ReduxDispatch<Action> & ThunkDispatch;
export type ThunkAction = (dispatch: Dispatch, getState: GetState) => any;
```